src/tests/ftest/container/snapshot.py

```diff
@@ -474,11 +474,11 @@ def test_snapshots(self):
 
         #(6)Destroy the individual snapshot
             self.log.info("=(6.%s)Destroy the snapshot epoch: %s",
-                          ss_number, epoch)
+                          ss_number, snapshot.epoch)
             try:
-                snapshot.destroy(coh, epoch)
-                self.log.info("  ==snapshot epoch %s successfully destroyed",
-                              epoch)
+                snapshot.destroy(coh, snapshot.epoch)
+                self.log.info("  ==snapshot.epoch %s successfully destroyed",
+                             snapshot.epoch)
```

Comment:
```suggestion
                self.log.info(
                    "  ==snapshot.epoch %s successfully destroyed", snapshot.epoch)
```
The second line should start after the `(` above, but I recommend making a line break right after the left paren. This is our preferred way because characters are distributed more consistently across a line.